support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/GuardianWeeklySubscriptionBuilder.scala

```diff
@@ -91,6 +92,16 @@ class GuardianWeeklySubscriptionBuilder(
     }
   }
 
+  private[this] def overrideReaderTypeIfRequired(state: GuardianWeeklyState): ReaderType = {
```

Comment:
actually, DigitalPack is the only Product that has a ReaderType specified up front
https://github.com/guardian/support-frontend/blob/34a01860ac4ad467d7a471528d6f0439324e21c6/support-models/src/main/scala/com/gu/support/workers/Products.scala#L35
so this function isn't overriding at all, it's just deciding it
```suggestion
  private[this] def readerTypeFromState(state: GuardianWeeklyState): ReaderType = {
```